src/pages/index.js

```diff
@@ -14,81 +16,190 @@ const addCardAddButton = document.querySelector(selectors.cardAddButton);
 const updateProfileButton = document.querySelector(selectors.profileEditButton);
 const cardFormElement = document.querySelector(selectors.cardPopup);
 const profileFormElement = document.querySelector(selectors.profilePopup);
+const avatarFormElement = document.querySelector(selectors.avatarPopup);
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms